### ISSUE TITLE
error whilst compiling

### ISSUE THREAD

**Issue by Toolfolks** [DelphiWorlds/Kastri#265]
Delphi 12.
Kastri from github
I have the demo group project running and getting various error. See image.

I seem to remember having to right click and revert defaults ( a few years ago) but that option is not there.

What do I need to do please.

PS the camera app worked ok.....
![BarcodeReader - Delphi 12 Community Edition - Unit1  Built  23_11_2024 00_12_23](https://github.com/user-attachments/assets/6efcdcb1-3ffc-470b-a19b-9877485ca5f9)

**Comment by DelphiWorlds** [comment]
Right-click the Libraries node - the "Revert to default" option is there.

**Comment by Toolfolks** [comment]
Got it thanks....
